python/lsst/ts/FiberSpectrograph/fiberSpectrograph.py

```diff
@@ -367,35 +396,52 @@ def to_str(value):
 @enum.unique
 class AvsReturnCode(enum.IntEnum):
     success = 0
-    invalidParameter = -1
-    operationNotSupported = -2
-    deviceNotFound = -3
-    invalidDeviceId = -4
-    operationPending = -5
-    timeout = -6
-    invalidPassword = -7
-    invalidMeasurementData = -8
-    invalidSize = -9
-    invalidPixelRange = -10
-    invalidIntegrationTime = -11
-    invalidCombination = -12
-    invalidConfiguration = -13
-    noMeasurementBufferAvailable = -14
-    unknown = -15
-    communicationError = -16
-    noSpectraInRam = -17
-    invalidDLLVersion = -18
-    noMemory = -19
-    dllInitialisationError = -20
-    invalidState = -21
-    invalidReply = -22
-    accessError = -24
-    invalidParameterNumberPixels = -100
-    invalidParameterADCGain = -101
-    invalidParameterADCOffset = -102
-    invalidMeasurementParameterAvgSat2 = -110
-    invalidMeasurementParameterAvgRam = -111
-    invalidMeasurementParameterSyncRam = -112
+    ERR_INVALID_PARAMETER = -1
+    ERR_OPERATION_NOT_SUPPORTED = -2
```

Comment:
I confess I'm glad to see the this name changing. Matching the header is a great idea and I _think_ our coding standards suggest uppercase for enum constants.